### PR TITLE
feat: api-client to get a cart

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
api-client to request the data of the cart from bigcommerce

## Related Issue
VBC-151

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/api-client/src/types/cart/responses.ts [hunk 1]
```diff
@@ -5,9 +5,15 @@ import { Cart } from '.';
  */
 export type AddLineItemsResponse = {
   data: Cart;
+  meta: Record<string, never>;
```
Comment (WojtekTheWebDev) [inline packages/api-client/src/types/cart/responses.ts:8]: Why `never`? Won't `unknown` be better?

**Review by WojtekTheWebDev — COMMENTED** [review]
LGTM! Just 1 question and it will be approved

File: packages/api-client/src/types/cart/responses.ts (hunk 1 above)
Comment (zoltantakacs-born) [reply packages/api-client/src/types/cart/responses.ts:8]: because it is an empty object

**Review by scernisevs — APPROVED** [review]
LGTM

**Review by WojtekTheWebDev — APPROVED** [review]
LGTM

**Pull request merged**